redact/redact.go

```diff
@@ -0,0 +1,59 @@
+package redact
+
+import (
+	"net/http"
+	"strings"
+)
+
+var SensitiveFields = []string{
+	"authorization",
+	"cookie",
+	"set-cookie",
+
+	"access_token",
```

Comment:
In a similar vein, why not replace all of these with simply `"token"`, which itself seems worth protecting.